js/signin.js

```diff
@@ -1,82 +1,104 @@
-import { $, SIGN_INFO_REGEX } from './common.js';
+import { $ } from './utils/dom.js';
+import { REGEX } from './constants/regex.js';
+import { KEY } from './constants/key.js';
+import { MESSAGE } from './constants/message.js';
+import { POST_API } from './constants/apiRequest.js';
+import { API_URL } from './constants/apiUrls.js';
```

Comment:
아래와 같이 import를 깔끔하게 처리해줄 수도 있는데요~
`barrel export`라는 키워드로 학습해주시면 좋을 것 같아요!
```suggestion
import { $ } from './utils/dom.js';
import { REGEX, KEY, MESSAGE, POST_API, API_URL } from './constants/index.js';
```